README.md

```diff
@@ -1,267 +1,189 @@
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- SUPPORT -->
-
-## ⭐️ Show your support <a name="support"></a>
-
-> Write a message to encourage readers to support your project
-
-If you like this project...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- ACKNOWLEDGEMENTS -->
-
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LICENSE -->
-
-## 📝 License <a name="license"></a>
-
-This project is [MIT](./LICENSE) licensed.
-
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 **BUDGET-APP** ](#-Budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [🚀 Live Demo](#live-demo)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+## ![My Expenses](https://cdn.vanguardngr.com/wp-content/uploads/2021/05/Easy-Budget.jpg)
+
+# 📖 [Expense Tracker] <a name="about-project">
+</a>
+
+This is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> This application is built with PostgreSQL as a Database.
+<a href="https://www.postgresql.org/" target="_blank" rel="noreferrer"> <img
+      src="https://user-images.githubusercontent.com/57408419/178756727-8bfad119-18c2-49eb-98be-8b2d4bd71dd6.jpeg"
+      alt="react" width="120" height="70" /> </a>
+
+<details>
+  <summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby - A Programmer's Best Friend</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Ruby on Rails</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[User identifcations]**
+- **[Can view all categories ]**
+- **[User can see how much spent in each each categorie]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://moneymanager-342g.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Video Presentation
+
+[video link](https://drive.google.com/file/d/16Qktky0anaig0xfyzDlhOH9ppj0Bn5Og/view?usp=sharing)

```

Comment:
- [ ] It is always advisable to make your links public rather than `private` to confirm whether the `shared` link is public or private, you can go to the `media` settings by right click on the content and following the options. To double-check things, you can also try to open the link in the `incognito` mode. In this way, you can make sure that your link or video is accessible to the person. Kindly fix the issue and make sure that everything is working fine before asking the review.

![image](https://github.com/hydaline-code/expense_tracker/assets/66476329/d3e64e60-86e3-4c35-aa31-46bb909b4f99)